spec/.eslintrc

```diff
@@ -7,7 +7,8 @@
     "array-bracket-spacing": ["off"],
     "no-sparse-arrays": ["off"],
     "object-property-newline": ["off"],
-    "no-underscore-dangle": ["off"]
+    "no-underscore-dangle": ["off"],
+    "no-unused-expressions": ["off"]
```

Comment:
I think it is better to leave no-unused-expressions turned on. We have configured dirty-chai which forces all the assertion 'properties' into functions. So expect(blah).to.be.undefined needs to be expect(blah).to.be.undefined(). This is to avoid common errors like expect(blah).to.be.defined - which does absolutely nothing but I have seen it often enough!

Reply:
Noted - I have removed this and updated test with `to.be.undefined()`